ovos_tts_plugin_server/__init__.py

```diff
@@ -14,12 +20,30 @@ def get_tts(self, sentence, wav_file, lang=None, voice=None):
         params = {"lang": lang, "voice": voice}
         if not voice or voice == "default":
             params.pop("voice")
-        data = requests.get(f"{self.host}/synthesize/{sentence}",
-                            params=params).content
+        if self.host:
+            if isinstance(self.host, str):
+                servers = [self.host]
+            else:
+                servers = self.host
+        else:
+            servers = self.public_servers            
+            random.shuffle(servers)  # Spread the load among all public servers
```

Comment:
Just a note that shuffling on every call could cause inconsistent behavior based on remote server availability and what version of a container it is running